starter-code/src/viking.js

```diff
@@ -1,11 +1,111 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health=health;
+    this.strength=strength;
+}
+
+Soldier.prototype.attack=function(){
+    return this.strength;
+}
+Soldier.prototype.receiveDamage=function(damage){
+    this.health-=damage;
+}
 
 // Viking
-function Viking() {}
+function Viking(name,health,strength) {
+    Soldier.call(this,health,strength);
+    this.name=name;
+    
+}
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
```

Comment:
You don't need this here. you already inherited the Solider class

Reply:
I remove it